packages/node_modules/@cerebral/fluent/tsconfig.json

```diff
@@ -2,13 +2,17 @@
   "compilerOptions": {
     "outDir": "./lib",
     "target": "es5",
-    "module": "es2015",
     "strict": true,
     "strictPropertyInitialization": false,
     "moduleResolution": "node",
     "declaration": true,
-    "declarationDir": "./",
-    "lib": ["es2015", "dom"]
+    "declarationDir": "./lib",
```

Comment:
i guess the default is outDir, so we can omit it

Reply:
Yep that's right.
Fixed.